packages/terra-data-grid/src/DataGrid.jsx

```diff
@@ -217,124 +180,75 @@ const DataGrid = injectIntl((props) => {
     props.focusFuncRef,
     () => ({
       setFocusedRowCol,
-      getFocusedCell() { return { row: focusedRow, col: focusedCol }; },
+      getFocusedCell() { return { row: focusedRow.current, col: focusedCol.current }; },
     }),
     // eslint-disable-next-line react-hooks/exhaustive-deps
     [focusedCol, focusedRow],
   );
 
-  // -------------------------------------
-  // callback Hooks
-
-  const gridRef = useCallback((node) => {
-    if (!node) {
-      return;
-    }
-
-    grid.current = node;
-
-    const resizeObserver = new ResizeObserver(() => {
-      // Update table height state variable
-      setTableHeight(grid.current.offsetHeight - 1);
-    });
-
-    // Register resize observer to detect size changes
-    resizeObserver.observe(node);
-  }, []);
-
   // -------------------------------------
   // useEffect Hooks
 
-  // useEffect for row selection
   useEffect(() => {
-    setDataGridColumns(displayedColumns.map((column) => initializeColumn(column)));
-  // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [hasSelectableRows]);
+    // We have to search the dom for the table ref because react-intl v2 does not support forwardRef.
```

Comment:
Discussed offline: See comment about react-intl 2 not supporting forwardRef (but supported in v3 which we currently do not support)